Assets/HoloToolkit/Utilities/Scripts/HeadsUpDirectionIndicator.cs

```diff
@@ -0,0 +1,122 @@
+﻿using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    // The easiest way to use this script is to drop in the HeadsUpDirectionIndicator prefab
+    // from the HoloToolKit. If you're having issues with the prefab or can't find it,
+    // you can simply create a UI->Image object and attach this script on the Image child
+    // of the Canvas. In that case you'll need to pick your own sprite. You'll also need to
+    // choose an appropriate "pivot" point on the RectTransform. After that you simply
+    // need to specify the "targetObject" and then you should be set.
+    // 
+    // This script takes for granted that your UI is rendering in Camera space and that the
+    // up vector is aligned with the "pointing" direction of your image.
+    [RequireComponent(typeof(RectTransform))]
+    public class HeadsUpDirectionIndicator : MonoBehaviour
+    {
+        [Tooltip("The object the direction indicator will point to.")]
+        public GameObject targetObject;
```

Comment:
Nit: Rename `IndicatorMarginPercentage` to `indicatorMarginPercentage`, make it private and add `[SerializeField]` attribute tag so they still show in Inspector window.